src/main/openapi/openapi_v1.yaml

```diff
@@ -227,10 +227,660 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/Collection'
+  
+# IIIF Presentation endpoints
```

Comment:
Could you add a few words about IIIF Presentation (with links to the standard, which version we support, how complete we are)? Maybe at the top of the OpenAPI YAML

Reply:
IIIF Presentation is now described in the top most description.

Reply:
The IIIF description is fine in itself, but the API is about more than "just" IIIF. Maybe mention the three formats under `/record/{id}` at the beginning of the presentation and make a second paragraph convering the IIIF-part?

...Okay, not the best suggestion. My point is primarily to avoid the impression that `ds-present` is only about IIIF.